src/Core/Silk.NET.Core/Native/Enums/D3DCBufferType.gen.cs

```diff
@@ -0,0 +1,38 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+
+using System;
+using Silk.NET.Core.Attributes;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.Core.Native
+{
+    [NativeName("Name", "_D3D_CBUFFER_TYPE")]
+    public enum D3DCBufferType
```

Comment:
now this really doesn't feel like a core type. it even starts with D3D

Reply:
Vulkan and OpenGL stuff are in Core. There has never been any guidelines for what constitutes a core addition, just whatever is common across some assemblies.

Reply:
I mean yes, perfectly fair point, though I think I've brought up that this is imo a problem too.
I wouldn't complain if it was one or two types, but these are like 10 generated types. Feels like it should just be a separate DX shared lib

Reply:
Consumers just don't want to have lots of assemblies. Again, more than happy to discuss a Silk.NET.Core.Slim for non-Silk.NET uses.